src/reference/changes.rst

```diff
@@ -36,6 +36,19 @@ Cylc 8.5
    :cylc-rose: `1.5 <https://github.com/cylc/cylc-rose/blob/master/CHANGES.md>`__
    :rose: `2.4 <https://github.com/metomi/rose/blob/master/CHANGES.md>`__
 
+More task information in the GUI (and Tui)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Task now icons in the GUI and TUI have modifier badges for tasks waiting for:
+
+* retries |task-retry|
+* xtriggers |task-xtriggered|
+* wallclock |task-wallclock|
+
+Xtrigger, wallclock trigger, retry trigger and task run mode are now
+shown in the information view:
+
+.. image:: ../img/info-view.xtriggers.png
```

Comment:
The other screenshots are all center aligned. You might want to fiddle the width to taste:

```suggestion
.. image:: ../img/info-view.xtriggers.png
   :align: center
   :width: 65%
```